pycqed/analysis/measurement_analysis.py

```diff
@@ -3746,14 +3746,18 @@ def run_default_analysis(self, print_fit_results=False,
             print(lmfit.fit_report(fit_res))
 
         fig, ax = self.default_ax()
-        textstr = '$f_{\mathrm{center}}$ = %.4f $\pm$ (%.3g) GHz' % (
-            fit_res.params['f0'].value, fit_res.params['f0'].stderr) + '\n' \
-            '$Qc$ = %.1f $\pm$ (%.1f)' % (fit_res.params['Qc'].value, fit_res.params['Qc'].stderr) + '\n' \
-            '$Qi$ = %.1f $\pm$ (%.1f)' % (
-                fit_res.params['Qi'].value, fit_res.params['Qi'].stderr)
 
-        # textstr = '$f_{\mathrm{center}}$ = %.4f $\pm$ (%.3g) GHz' % (
-        #     fit_res.params['f0'].value, fit_res.params['f0'].stderr)
+        if 'hanger' in fitting_model:
```

Comment:
I added tests to the analyses I changed or added. The plots look like this:
![heterodyne frequency_heterodyne frequency hz _vs_mag](https://cloud.githubusercontent.com/assets/1426628/23364244/937be89c-fcfe-11e6-86cb-5d627e9e9fed.png)
![acquisition delay_delay_vs_amplitude](https://cloud.githubusercontent.com/assets/1426628/23364257/9db5c986-fcfe-11e6-9222-171cf868fb50.png)